Source/Core/VideoBackends/Software/EfbInterface.cpp

```diff
@@ -428,17 +424,17 @@ void BlendTev(u16 x, u16 y, u8* color)
   {
     Dither(x, y, dstClrPtr);
     if (bpmem.blendmode.alphaupdate)
-      SetPixelAlphaColor(offset, dstClrPtr);
+      SetPixelAlphaColor(offset, dstClr);
     else
-      SetPixelColorOnly(offset, dstClrPtr);
+      SetPixelColorOnly(offset, dstClr);
   }
   else if (bpmem.blendmode.alphaupdate)
   {
-    SetPixelAlphaOnly(offset, dstClrPtr[ALP_C]);
+    SetPixelAlphaOnly(offset, dstClr);
```

Comment:
Are you sure this yields the correct result if you drop the indexing expression of `ALP_C` there?